packages/react/src/analytics/integrations/Vitorino/Vitorino.js

```diff
@@ -71,28 +60,30 @@ export default class Vitorino extends integrations.Integration {
   constructor(options = {}, loadData, analytics) {
     super(options, loadData, analytics);
 
-    this.isScriptLoaded = false;
+    const baseOptions = omit(options, ['riskified', 'forter']);
 
-    this.getEventsMapper = null;
+    baseOptions.environment = getEnvironmentFromOptions(options);
 
-    this.isVitorinoConfigured = false;
+    // load default if not set
+    options.activeIntegrations ??= [
+      VITORINO_PROVIDERS.forter,
+      VITORINO_PROVIDERS.riskified,
+    ];
 
-    this.vitorinoTrackCallback = null;
-
-    this.initializePromiseResolve = null;
-
-    this.userIdPromiseResolve = null;
-
-    this.initializePromise = new Promise(resolve => {
-      this.initializePromiseResolve = resolve;
-    });
-
-    this.userIdPromise = new Promise(resolve => {
-      this.userIdPromiseResolve = resolve;
-    });
+    if (options.activeIntegrations.includes(VITORINO_PROVIDERS.forter)) {
```

Comment:
Since we're calling `.includes()` on the `options.activeIntegrations` we should ensure the developer passes a valid array, otherwise this will throw an error if an object is passed instead (for example).
Can you add a validation of some sort here, please?